src/main/java/io/openliberty/tools/intellij/runConfiguration/LibertyRunSettingsEditor.java

```diff
@@ -27,7 +26,7 @@
 public class LibertyRunSettingsEditor extends SettingsEditor<LibertyRunConfiguration> {
 
     private JPanel root;
-    private LabeledComponent<EditorTextField> editableParams;
+    private JTextField editParam;
```

Comment:
Can we use EditorTextField (com.intellij.ui.EditorTextField) instead of JtextField. As it have additional properties than JTextField.

Reply:
Changed the JtextField to EditorTextField.